install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -56,40 +86,102 @@ mkdir $HOME/.local/share/odemis/log 2>/dev/null
-    printf "$(date) Parsing file: $fn\n" >> $parse_logfile
-    # extract the microscope name by using this small Python script
+process_file() {
+    local fn="$1"
+
+    # Add a new header to the logfile
+    printf "$(date) Parsing file: $fn\n" >> "$parse_logfile"
+
+    # Extract the microscope name using a small Python script
     mic_name="$(python3 -c "import sys; import yaml; from odemis.odemisd import modelgen; \
         f = open('$fn'); \
         d = yaml.load(f, modelgen.SafeLoader); \
-        print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))" 2>>$parse_logfile)"
+        print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))" 2>>"$parse_logfile")"
+
     mic_list+=("$fn")
+
     if test "$mic_name" == ""; then
         # save the file name in the error list
         error_filenames+=("$(basename ${fn})")
         mic_name=">>ERROR parsing $(basename ${fn})"
         # add a newline to the logfile
         echo "" >> $parse_logfile
     fi
+
     mic_list+=("$mic_name")
-done
 
-# prompt the user with a list of microscope configurations to choose from
-selection="$(zenity --list --title="Odemis configuration starter" \
-    --list --text "Please select a configuration to start:" \
-    --column "" --column "" "${mic_list[@]}" --hide-header \
-    --width=300 --height=400 \
-    --hide-column 1 --print-column=1 2>/dev/null)"
+    # Format the setup name
+    setup_name="${fn#*-}"
+    setup_name="${setup_name%.odm.yaml}"
+    mic_list+=("$setup_name")
+}
+
+# Check if the input is a directory or a wildcard
+if [ -d "${args[0]}" ]; then
+    # Input is a directory, list YAML files and extract unique prefixes
+    yaml_files=("${args[0]}"/*.yaml)
+
+    if [ ${#yaml_files[@]} -eq 0 ]; then
+        echo "No YAML files found in the directory."
+        exit 1
+    fi
+
+    # Extract unique prefixes
+    prefixes=()
+    for fn in "${yaml_files[@]}"; do
+        base_name="$(basename "$fn")"
+        prefix="${base_name%%-*}"
+        if [[ ! " ${prefixes[@]} " =~ " ${prefix} " ]]; then
+            prefixes+=("$prefix")
+        fi
+    done
+
+    # Prompt user to select a prefix
+    selected_prefix=$(zenity --list --title="Odemis configuration starter" \
+        --text="Please select a system:" \
+        --column="System" "${prefixes[@]}" \
+        --width=400 \
+        --height=500
+    )
+
+    if test "$selected_prefix" == ""; then
+        echo "No selection made. Exiting."
+        exit 1
+    fi
+
+    # Process files matching the selected prefix
+    for fn in "${args[0]}/$selected_prefix"*.yaml; do
+        process_file "$fn"
+    done
+else
+    # Input is a wildcard or single file, process directly
+    for fn in "${args[@]}"; do
+        process_file "$fn"
+    done
+fi
+
+selection=$(zenity --list --title="Odemis configuration starter" \
+    --text "Please select a configuration to start:" \
+    --column="Configuration" --column="System" --column="Variant" \
```

Comment:
Please don't change the display of this window. There are several systems which uses it, and it doesn't make sense to have 3 columns. The idea is that the selection is very simple: just one name per row.

If the problem is that in the case of the simulators, they all have identical names, then we should just fix the names of the simulator microscope files. So far, this name wasn't useful, so we never cared. That's the only reason.

Reply:
With name, do you mean the "SPARC2-Fiber" part in this snippet from  `sparc2-ext-spec-sim-cntpmt.odm.yaml`?
```
SPARC2-Fiber: {
    class: Microscope,
    role: sparc2,
    children: ["SEM E-beam", "SEM Detector","Calibration Light", "Integrated Spectrograph",
               "Optical Path Properties", "Spectrometer Selector", "External Spectrograph",
               "Camera", "Spectrometer", "Fiber align", "Spectral Camera",
               "Spec CCD Flipper", "Spectrograph focus", "Monochromator",
               "Mirror Actuators", "Mirror Actuators in XY referential", "Slit",
               "Lens1 Mover", "Lens2 Switch"],
}
```

Reply:
I will leave it as is, and will ask Apps Specialists to come up with a better name (will be out of scope for now).